tests/tests.rs

```diff
@@ -1,7 +1,7 @@
 extern crate diff;
 extern crate is_executable;
 
-use is_executable::is_executable;
+use is_executable::{ is_executable, is_permitted };
```

Comment:
This import won't work when compiling for not-Unix. Importing `is_permitted` must be done in `mod unix`